roles/cifmw_cephadm/tasks/dashboard/configure_dashboard_backends.yml

```diff
@@ -34,5 +34,5 @@
   ansible.builtin.command: |
     {{ cifmw_cephadm_ceph_cli }} config set \
     mgr mgr/dashboard/{{ current_mgr }}/server_addr \
-    {{ hostvars[item][all_addresses] | ansible.utils.ipaddr(cifmw_cephadm_rgw_network) | first }}
+    {{ hostvars[item][all_addresses] | ansible.utils.ipaddr(cifmw_cephadm_monitoring_network) | first }}
```

Comment:
+1 thanks

Reply:
+1  considering the changes to `cifmw_cephadm_rgw_network` in #2301, where RGW can use a saperate nework. 
this is very much needed here.